mods/ts/rules/defaults.yaml

```diff
@@ -227,13 +227,21 @@
 		TargetTypes: Ground, Infantry
 	RenderSprites:
 	WithInfantryBody:
-	WithDeathAnimation:
+	WithDeathAnimation@normal:
 		DeathTypes:
 			BulletDeath: 1
 			SmallExplosionDeath: 2
-			ExplosionDeath: 3
-			InfantryExplodeDeath: 4 #not used by any warhead; TODO: check what should use this type and add it!
+			HeadshotDeath: 3 #not used by any warhead; TODO: check what should use this type and add it!
```

Comment:
OK, can we replace "TODO: check what should use this type and add it!" with "TODO: should be used by/for X and Y"?